dotcom-rendering/src/components/KeyEventsCarousel.stories.tsx

```diff
@@ -34,47 +33,47 @@ const Wrapper = ({ children }: { children: React.ReactNode }) => {
 	);
 };
 
-const SingleKeyEventCarousel = () => {
+export default {
+	component: KeyEventsCarousel,
+	title: 'Components/KeyEventsCarousel',
+};
+
+export const SingleKeyEventCarousel = () => {
 	return (
 		<Wrapper>
 			<KeyEventsCarousel
 				keyEvents={SingleKeyEvent}
 				filterKeyEvents={false}
-				format={format}
 				id="key-events-carousel-desktop"
 			/>
 		</Wrapper>
 	);
 };
-const ShortKeyEventCarousel = () => {
+SingleKeyEventCarousel.storyName = 'Single Key Event Carousel';
+SingleKeyEventCarousel.decorators = [splitTheme()];
```

Comment:
Does it make sense to use the default formats? A lot of them have `Design` that is neither `LiveBlog` nor `DeadBlog`

Reply:
Good point, I'll update...